Automattic/newspack-theme
# Removes footer navigation CSS

### All Submissions:


### Changes proposed in this Pull Request:

This PR removes the footer menu CSS; the menu itself was removed in #108.

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`.
2. Confirm there are no build errors.

### Other information:
